src/components/Sep6Send.tsx

```diff
@@ -0,0 +1,183 @@
+import React, { useEffect, useState } from "react";
+import { useDispatch } from "react-redux";
+import { Button, InfoBlock, Input, Select } from "@stellar/design-system";
+import { Heading2 } from "components/Heading";
+import { Modal } from "components/Modal";
+import { TextLink } from "components/TextLink";
+import { resetActiveAssetAction } from "ducks/activeAsset";
+import {
+  resetSep6DepositAction,
+  submitSep6DepositFields,
+  sep6DepositAction,
+} from "ducks/sep6DepositAsset";
+import { useRedux } from "hooks/useRedux";
+import { ActionStatus } from "types/types.d";
+
+export const Sep6Send = () => {
+  const { sep6DepositAsset } = useRedux("sep6DepositAsset");
+  const { depositResponse } = sep6DepositAsset;
+
+  const [formData, setFormData] = useState<any>({
+    depositType: {},
+    fields: {},
+  });
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (sep6DepositAsset.status === ActionStatus.NEEDS_INPUT) {
+      setFormData({
+        depositType: {
+          type: sep6DepositAsset.data.depositTypes.type.choices[0],
```

Comment:
default formData to the first `choice` as some assets (like SRT) will only have one